src/ui/Version.ts

```diff
@@ -106,7 +106,7 @@ export class OvaleVersionClass {
             if (channel) {
                 this.module.SendCommMessage(MSG_PREFIX, message, channel);
             }
-            self_timer = this.module.ScheduleTimer("PrintVersionCheck", 3);
+            self_timer = this.module.ScheduleTimer(this.PrintVersionCheck, 3, this);
```

Comment:
If PrintVersionCheck is declared as a lambda `PrintVersionCheck = () => { ... }`  it isn't necessary to add this argument.

Reply:
I wouldn't mind changing this file quite a bit more. It looks like the original Version.lua file translated straight into TypeScript, but I believe it could be written in a nicer way now. I could work on this to learn more TypeScript if you don't mind some time on fixing this.